repo-scripts/api-documenter/src/cli/TocAction.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  CommandLineFlagParameter,
+  CommandLineStringParameter
+} from '@rushstack/ts-command-line';
+import { ApiDocumenterCommandLine } from './ApiDocumenterCommandLine';
+import { BaseAction } from './BaseAction';
+import { generateToc } from '../toc';
+
+export class TocAction extends BaseAction {
+  private _g3PathParameter!: CommandLineStringParameter;
+  private _jsSDKParameter!: CommandLineFlagParameter;
+  public constructor(parser: ApiDocumenterCommandLine) {
+    super({
+      actionName: 'toc',
+      summary: 'Generate documentation as Markdown files (*.md)',
+      documentation:
+        'Generates API documentation as a collection of files in' +
+        ' Markdown format, suitable for example for publishing on a GitHub site.'
+    });
+  }
+
+  protected onDefineParameters(): void {
+    super.onDefineParameters();
+
+    this._g3PathParameter = this.defineStringParameter({
+      parameterLongName: '--g3-path',
```

Comment:
How about calling this `--host-path` or `--site-dir` (something that's not g3 specific)?

Reply:
changed to `--host-path`